### PR TITLE
[11.0][UPD] travis.yml

### PR DESCRIPTION
**Pull request by MiquelRForgeFlow**
Greenify repo.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -28,13 +29,9 @@ env:
   - TRANSIFEX="1"
```
Comment (pedrobaeza) [inline .travis.yml]: Adapt this part also

Comment (MiquelRForgeFlow) [reply .travis.yml]: done

**Review by pedrobaeza — APPROVED** [review]

**Pull request merged**